README.md

```diff
@@ -0,0 +1,59 @@
+# zabbix-aruba-sensor-monitoring
+
+This python module provides Zabbix monitoring support for Aruba sensors using the Aruba User Experience Insight API. This API is currently in closed beta state and subjective to change at any time.
+
+
+
+## Requirements
+
+- Zabbix agent
+- pip
+- requests (installed automatically as dependency)
+
+
+
+## Installation
+
+1. Install the python module using pip.
+
+```
+pip install https://github.com/digiaiiris/zabbix-aruba-sensor-monitoring/releases/download/1.0.0/aruba-sensor-monitoring-1.0.0.tar.gz
+```
+
+2. Copy the [Zabbix agent configuration](etc/zabbix/zabbix_agent.d/ic_aruba.conf) to /etc/zabbix/zabbix_agent.d directory.
```

Comment:
Wrong path, directory is zabbix_agentd.d (one d is missing)

Reply:
What a typo. Good catch. Fixed in commit 50fd222242a14caa0b8d9627e38e233fcebcb0f6.